src/SignaturePad.tsx

```diff
@@ -358,7 +358,7 @@ class SignaturePad extends React.PureComponent<Props, State> {
         let data;
 
         if (this.props.redrawOnResize && this.signaturePad) {
```

Comment:
```suggestion
        if (this.props.redrawOnResize && this.signaturePad && !this.signaturePad.isEmpty()) {
```

Reply:
I'd prefer to include the extra condition on whether to set the `data` variable in the existing `if`. I think this is more readable than using a 2nd inline condition.